stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -65,6 +67,8 @@
   private ChildNodeRemovedEvent mCachedChildNodeRemovedEvent;
   private ChildNodeInsertedEvent mCachedChildNodeInsertedEvent;
 
+  private HashMap<String, List<Integer>> mSearchResults;
```

Comment:
Also, we can probably go ahead and `clear()` this in `onLastPeerUnregistered()`.